src/app/assumptions-and-limitation/assumptions-and-limitation.component.ts

```diff
@@ -85,14 +84,4 @@ export class AssumptionsAndLimitationComponent implements AfterContentInit {
   public ngAfterContentInit(): void {
     this.assumptionsLimitation.nativeElement.focus({ preventScroll: true });
```

Comment:
I dont think this is needed any more. As was added due to the A&L being a popup

Reply:
Good shout, I think it also related to the form potentially being at the bottom when the assumptions button was clicked, which wasn't possible since it moved above the form.

Reply:
Hopefully resolved now